services/core/cards-publication/src/main/modeling/swagger.yaml

```diff
@@ -700,19 +677,37 @@ paths:
   /cards:
     post:
       tags:
-        - cards
+        - card
         - creation
       summary: publish cards
-      description: Publish one or more card to OperatorFabric. Be careful, to know if the cards were published well, then it is necessary to consult the report returned in the body of the request (do not rely only on the response code).
-      operationId: publishCards
+      description: Publish one card to OperatorFabric - Deprecated endpoint (use /cards/card instead)
```

Comment:
Why is it necessary to change /cards to /cards/card? Couldn't we keep /cards and advertise the fact that now it only accepts one card? Projects that used to send several cards will have to update their code regardless of the name of the endpoint.
In addition, what bothers me is that if posting cards is managed via /cards/card, it won't be coherent with the corresponding DELETE and GET operations that are still attached to the /cards endpoint, right?

Reply:
you are right , i was wrong about the naming convention , thinking that as we post a card it shall be singular. After having a look on the net , the good way is to POST /cards and not /cards/Card as you said . 
I will correct it